src/app/shared/ui/paragraphs/paragraphs.component.ts

```diff
@@ -17,6 +17,7 @@ import { SplitLinesPipe } from '@app/shared/pipes/split-lines.pipe';
 })
 export class ParagraphsComponent {
   text = input.required<string>();
-  header = input<string>();
+  label = input<string>();
+  computedLabel = computed(() => (this.label() ? `${this.label()}: ` : undefined));
```

Comment:
i would call it something more descriptive like  `labelWithColon` or `firstParagraphPrefix`